docs/requests.md

```diff
@@ -55,6 +55,16 @@ Headers are exposed as an immutable multi-dict.
 
 For example: `request.query_params['abc']`
 
+#### Cookies
+
+Cookies are exposed as an immutable dict.
```

Comment:
It's not actually immutable at the moment, tho it'd be nice if it was.
I'll merge this as-is and then update this.